Google.PowerShell.IntegrationTests/packages.config

```diff
@@ -0,0 +1,16 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Google.Api.Gax" version="1.0.0-beta12" targetFramework="net452" />
+  <package id="Google.Api.Gax.Rest" version="1.0.0-beta12" targetFramework="net452" />
+  <package id="Google.Apis" version="1.20.0" targetFramework="net452" />
+  <package id="Google.Apis.Auth" version="1.20.0" targetFramework="net452" />
+  <package id="Google.Apis.Bigquery.v2" version="1.20.0.667" targetFramework="net452" />
+  <package id="Google.Apis.Container.v1" version="1.20.0.662" targetFramework="net452" />
+  <package id="Google.Apis.Core" version="1.20.0" targetFramework="net452" />
+  <package id="Google.Cloud.BigQuery.V2" version="1.0.0-beta09" targetFramework="net452" />
+  <package id="log4net" version="2.0.3" targetFramework="net452" />
+  <package id="Newtonsoft.Json" version="7.0.1" targetFramework="net452" />
+  <package id="System.Interactive.Async" version="3.1.1" targetFramework="net452" />
+  <package id="System.Net.Http" version="4.3.0" targetFramework="net452" />
+  <package id="Zlib.Portable.Signed" version="1.11.0" targetFramework="net452" />
+</packages>
```

Comment:
I'm not sure if we need packages.config for the PowerShell.IntegrationTests project?